src/rpdk/core/contract/resource_client.py

```diff
@@ -426,7 +444,12 @@ def generate_update_example(self, create_model):
             if self.create_only_paths:
                 self.validate_update_example_keys(unique_identifiers, update_example)
             update_example.update(unique_identifiers)
-            return update_example
+            create_model_with_read_only_properties = (
+                prune_properties_which_dont_exist_in_path(
```

Comment:
We are successfully including top level read only properties what about sub-properties which are populated with default values post creation?

Reply:
Spoke offline. Will be addressed in another PR.